src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
+        if (bibtexFields.isEmpty()) {
+            return false;
+        }
+
+        Optional<String> content = Optional.empty();
+        Optional<String> entryContent = Optional.empty();
+        for (String field : bibtexFields) {
+            if (BibEntry.TYPE_HEADER.equals(field) || "bibtextype".equals(field)) {
+                return false;
+            } else {
+                entryContent = entry.getFieldOptional(field);
+                content = BibDatabase.getResolvedField(field, entry, database.orElse(null));
```

Comment:
Oh. OK. Maybe just rename `entryContent` -> `plainFieldContent` and `content` to `resolvedFieldContent`.